utility/importer/chrome_importer.cc

```diff
@@ -196,7 +218,9 @@ void ChromeImporter::ImportBookmarks() {
   base::FilePath bookmarks_path =
     source_path_.Append(
       base::FilePath::StringType(FILE_PATH_LITERAL("Bookmarks")));
-  base::ReadFileToString(bookmarks_path, &bookmarks_content);
+  CreateCopyFile copy_bookmark_file(bookmarks_path);
+  base::ReadFileToString(copy_bookmark_file.copied_file_path(),
```

Comment:
same as above, but I'm not sure what happens if you pass an empty string to JSONReader